etna/metrics/functional_metrics.py

```diff
@@ -42,10 +66,16 @@ def mape(y_true: ArrayLike, y_pred: ArrayLike, eps: float = 1e-15) -> float:
 
     y_true_array = y_true_array.clip(eps)
 
-    return np.mean(np.abs((y_true_array - y_pred_array) / y_true_array)) * 100
+    mode_enum = FunctionalMetricMode(mode)
```

Comment:
Can't we somehow avoid such code duplication? May be this enum can return correct value for an axis parameter?

Reply:
We could do this, but I'm not sure that it is applicable to any case. I could simplify it the other way may be and choose the `axis` parameter in this cycle.